myst.yml

```diff
@@ -16,15 +16,14 @@ project:
     - pattern: "0*.md"
     - title: "Modules"
       children:
-        - pattern: "modules/0*/index.md"
+        - pattern: "modules/0*/index.*"
```

Comment:
This change will conflict with the change we just merged from #24 ; could you rebase? :pray: